### ISSUE TITLE
Can transboost work for the regression problem?

### ISSUE THREAD

**Issue by 799877006** [yihengsun/TransBoost#3]
Hi @yihengsun:

  I find there is only Class TransBoostClassifier in TransBoost. Is it possible to extend the TransBoost framework for regression problems. Maybe it will meet with some difficultity when calculate joint probability?

Thank you.

**Comment by yihengsun** [comment]
I agree with you. Theoritically, the algorithm can be extended to regression by replacing the ration of condition distribution with h_t(x)/h_s(x). But the ration may be extremely small or large sometimes in regression problems. And it's likely to adversely affect performance of the algorithm. So I think it won't work well in practice.
